### PR TITLE
Added section about boolean variables

### LINKED ISSUE #78559: Add a section about boolean variables
### Summary

Ansible accepts a broad range of values for boolean variables: `true/false`, `1/0`, `yes/no`, `True/False`, and so on. We recently reached consensus about representing all Boolean values as `true/false` in the module documentation generated from python docstrings - see [this vote](https://github.com/ansible-community/community-topics/discussions/120) and [this issue](https://github.com/ansible-community/community-topics/issues/116) and [this PR](https://github.com/ansible-community/antsibull-docs/pull/19). We plan to update examples and other mentions of boolean values in the docs over the next few months. 

The docs should also show users that they do not have to use `true/false` if they choose not to. Adding a section to the [documentation on variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#playbooks-variables) that explains all this will help users.

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/user_guide/playbooks_variables.rst

### Ansible Version

```console
N/A
```

### Configuration

```console
N/A
```

### OS / Environment

N/A

### Additional Information

None

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by wnjuguna**
##### SUMMARY

Fixes #78559 - Documents the list of supported boolean values. 

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst [hunk 1]
```diff
@@ -95,6 +95,26 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
...
            app_path: "{{ base_path }}/22"
 
+.. _boolean_variables:
+
+Boolean variables
+=================
+
+Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False``, and so on.
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+
```
Comment (bcoca) [inline docs/docsite/rst/playbook_guide/playbooks_variables.rst:105]: note that the strings will match is case insensitive so `'yes' == 'Yes' == 'yEs' == ... ` you get the idea

File: docs/docsite/rst/playbook_guide/playbooks_variables.rst [hunk 2]
```diff
@@ -95,6 +95,26 @@ You will see: ``ERROR! Syntax Error while loading YAML.`` If you add quotes, Ans
...
+.. table::
+   :class: documentation-table
+
+   ==================================================================================== ====================================================================
+    Valid values                                                                         Description
+   ==================================================================================== ====================================================================
+   ``true`` , ``True`` , ``t`` , ``yes`` , ``y`` , ``on`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+
+   ``false`` , ``False`` , ``f`` , ``no`` , ``n`` , ``off`` , ``'0'`` , ``0`` , ``0.0``   Falsy values
```
Comment (bcoca) [inline docs/docsite/rst/playbook_guide/playbooks_variables.rst]: `True` and `False` in this context are 'actual boolean values' while `'true'` and `'false'` are strings (why there is a duplication).

**Comment by wnjuguna** [comment]
ready_for_review

**Review by bcoca — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
@wnjuguna thanks for the ansible Docs fix and welcome to Ansible!

**Comment by wnjuguna** [comment]
@samccann Thank you!

**Pull request merged**